multi/stm32l4-multi/flash.h

```diff
@@ -15,14 +15,34 @@
 #define _FLASH_H_
 
 #include <stdint.h>
-#include "common.h"
-#include "config.h"
 
 #define FLASH_PAGE_SIZE         2048
 #define FLASH_OB_1_ADDR         0x1fff7800
 #define FLASH_OB_2_ADDR         0x1ffff800
 #define FLASH_OB_SIZE           16
 
+#ifndef FLASH_PROGRAM_1_ADDR
+#define FLASH_PROGRAM_1_ADDR 0x08000000
+#endif
+
+#ifndef FLASH_PROGRAM_2_ADDR
+#define FLASH_PROGRAM_2_ADDR 0x08080000
+#endif
+
+#ifndef FLASH_PROGRAM_BANK_SIZE
+#define FLASH_PROGRAM_BANK_SIZE (320 * 1024)
```

Comment:
I know that it's not a topic of this change but you sure that's valid value? 

Shouldn't be 512?

Reply:
Yes, fixed